core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -1180,6 +1185,17 @@ abstract class Record extends Aggregate {
 
   def elements: SeqMap[String, Data]
 
+  private[chisel3] lazy val _elements: SeqMap[String, Data] = {
```

Comment:
It would be good to have a comment describing why we use this now instead of `elements`.